### ISSUE TITLE
stream element

### ISSUE THREAD

**Issue by reschke** [interim-rfc-series-project-manager/draft-irse-xml2rfcv3-implemented#8]
There are multiple issues here.

1. According to xml2rfc's docs, it replaces and deprecates the stream *attribute* on \<seriesInfo>.
2. The stream attribute is there as well.
3. It's not clear what it is used for; it just seems to be invisible metadata.

And for the actual definition:

1. It's documented to be optional on \<reference> (ok)
2. But then the text content itself is optional as well (so an empty element would be ok, but we do not say what this means)
3. Constraining text content by an enumerated list is a first in the language, but we may want to reconsider why baking the allowed values into the grammar is a good idea.

My proposal would be to revisit why this attribute/element even exists.

**Comment by reschke** [comment]
In particular, it seems to be weird to have this as generic child element of \<reference>, when it only applies to RFCs and Internet Drafts.

**Comment by reschke** [comment]
It (the element variant) also does not seem to be used in published RFCs (they do use seriesInfo/@stream).

**Comment by jrlevine** [comment]
I agree this never should have been part of `<reference>` in the first place. Don't recall what problem it was supposed to address.

**Comment by reschke** [comment]
Maybe it was Henrik attempting to restore sanity in \<seriesInfo>. But of course the simpler solution is to juet keep using @submissionType on \<rfc>.
